templates/zerver/social_auth_select_email.html

```diff
@@ -8,22 +8,51 @@ <h1 class="get-started">Select account</h1>
         {% endtrans %}
     </div>
 
-    <div>
+    <div class="white-box">
         <form method="post" class="form-horizontal" action="/complete/{{ backend }}/">
-            <div class='choose-email-box'>
+            <div class="choose-email-box">
                 <input type="hidden" name="email" value="{{ primary_email }}" />
-                <button type="submit" >
-                    {{ primary_email }}
-                </button>
+                <div>
+                    <div class="email">
+                        {{ primary_email }}
+                    </div>
+                    <div class="paragraph">
+                        GitHub primary
+                        {% if avatar_url_primary %}
+                        - Existing account
+                        {% else %}
+                        - Create new account
+                        {% endif %}
+                    </div>
+                </div>
+                {% if avatar_url_primary %}
+                <div class="zulip_profile_picture">
+                    <img src="{{ avatar_url_primary }}" alt=""/>
+                </div>
+                {% endif %}
             </div>
         </form>
         {% for email in verified_non_primary_emails %}
         <form method="post" class="form-horizontal" action="/complete/{{ backend }}/">
-            <div class='choose-email-box'>
+            <div class="choose-email-box">
                 <input type="hidden" name="email" value="{{ email }}" />
-                <button type="submit" >
-                    {{ email }}
-                </button>
+                <div>
+                    <div class="email">
+                        {{ email }}
+                    </div>
+                    <div class="paragraph">
```

Comment:
Why not just make this a `p` element and change the margin/font-size accordingly instead of making a div with a new class?